pytest/tests/sanity/split_storage.py

```diff
@@ -76,6 +76,126 @@ def _check_split_storage_info(
         else:
             self.assertIsNone(cold_head_height)
 
+    def _migrate_to_split_storage(self, rpc, archival, archival_dir,
```

Comment:
Add a comment that gives a high-level explanation of what's this test does.
The PR has a good such description.